supported_cli_versions.json

```
@@ -0,0 +1,8 @@
+[
+  "v2.11.5",
```

```suggestion
  "v2.11.6",
```

(update since this PR was opened!)

---

This should hopefully force the required `CLI Test (v2.11.6)` checks to run